packages/fast-jss-manager-react/src/jss-manager.tsx

```diff
@@ -81,6 +81,13 @@ export class JSSManager<S, C> extends React.Component<JSSManagerProps<S, C>, JSS
         this.state = state;
     }
 
+    /**
```

Comment:
You can probably comment out line 78 also - that is where the update cycle got moved to.